商品詳細ページサーバサイド

y0913 opened the pull request:
# what
商品詳細ページを作成いたしました。
テストコードに関してましては後ほど記載して再度プルリクエスト を提出したいと思います。

# What to Improve
インデントを修正いたしました。
Createアクションについては、私が作成してしまいましたが、実際は担当街ということをチーム内で共有いたしました。そのため、担当者が修正の上、再度プルリクエスト を提出することになります。
お手数かけますがよろしくお願いいたします。

![localhost_3000_items_1](https://user-images.githubusercontent.com/44940979/50326257-b8e18c80-052c-11e9-9b19-8a601b0024ff.png)

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master'

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master' into 商品詳細ページサーバサイド

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master' into 商品詳細ページサーバサイド

y0913 committed:
Edit items_controller.rb

items_controller.rbに@items = Item.newを追記しました。

y0913 committed:
Edit item-details.scss

Scssファイルを編集しました。

y0913 committed:
Edit show.html.haml

Link_toを条件分岐しました。

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master' into 商品詳細ページサーバサイド

y0913 committed:
Edit item-details.scss & show.html.haml

tem-details.scss & show.html.hamlを編集しました。

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master' into 商品詳細ページサーバサイド

y0913 committed:
Merge remote-tracking branch 'abeshi1019/master' into 商品詳細ページサーバサイド

y0913 committed:
Edit db/schema.rb

エラー解決のため。

y0913 committed:
Edit色々

y0913 committed:
Edit my member's edit

メンバーの変更がgithubに反映されたのでコミットします。

y0913 committed:
Edit

y0913 committed:
Made item-detail page

商品詳細ページを作成いたしました。

miyagit commented on db/schema.rb:
```diff
@@ -88,18 +88,29 @@
     t.integer "size_id"
+<<<<<<< HEAD
+<<<<<<< HEAD
+>>>>>>> abeshi1019/master
```
コンフリクトは直しましょう。

y0913 committed:
Improve conflict

コンフリクトを解消しました。

miyagit commented on app/controllers/items_controller.rb:
```diff
@@ -6,6 +6,17 @@ def index
     render :new, layout: "sub-layout"
   end
+  def create
+  	Item.create(image: items_params[:image], text: items_params[:text], user_id: current_user.id)
```
引数をitem_paramsだけで出来るようにしましょう。

miyagit reviewed (commented):
インデントがずれているところがあるので、インデントを全て修正して下さい。

y0913 committed:
Edit インデント

インデントを修正いたしました。

Createアクションについては、私が作成してしまいましたが、実際は担当街ということをチーム内で共有いたしました。
そのため、担当者が修正の上、再度プルリクエスト を提出することになります。
お手数かけますがよろしくお願いいたします。
